shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -396,7 +396,9 @@ - (void)listenForMetaModifiedKeyUpEvents {
 }
 
 - (void)configureTrackingArea {
-  NSAssert(self.viewLoaded, @"View must be loaded before setting tracking area");
+  if (!self.viewLoaded) {
+    return;
```

Comment:
What causes this to get set again when the view loads?

Reply:
`- (void)viewDidLoad;` will call the `configureTrackingArea` selector

Reply:
cool - maybe just mention that here in a comment? otherwise LGTM